media-plugins/distrho-ports/distrho-ports-99999999.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3
+
+DESCRIPTION="Linux ports of Distrho plugins."
+HOMEPAGE="http://distrho.sourceforge.net"
+EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports.git"
+KEYWORDS=""
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="lv2 vst"
+REQUIRED_USE="|| ( lv2 vst )"
+
+RDEPEND="media-libs/alsa-lib
+  media-libs/freetype
+  media-libs/ladspa-sdk
+  media-libs/mesa
```

Comment:
You're right :)
I've updated the ebuilds to use `virtual/opengl` instead of `media-libs/mesa`